src/_pytest/terminal.py

```diff
@@ -440,7 +446,7 @@ def pytest_runtest_logreport(self, report):
                     self._write_progress_information_filling_space()
             else:
                 self.ensure_newline()
-                self._tw.write("[%s]" % rep.node.gateway.id)
+                self._tw.write("[%s]" % rep.node.gateway.id)  # type: ignore
```

Comment:
Would be nice to have a short comment on why the `type: ignore` is needed here. This why we'll be able to tell later whether it's a "TODO" or a "workaround mypy" thing.

Reply:
It's a workaround-xdist thing (I assume that xdist just puts it there).

Reply:
I'll keep it as is (it is somehow obvious from "xdist" in there, and I do not like extra comments for this above).  Also it might get reported when not being used/needed later anymore (there's a mode in mypy to do so).